extensions/wikia/Forum/ForumController.class.php

```diff
@@ -169,7 +168,7 @@ public function boardThread() {
 			$name = $lastReply->getUser()->getName();
 		}
 
-		if ( $lastReply->getUser()->isAnon() ) {
+		if ( $lastReply->getUser()->getId() == 0 ) {// anynymous contributor
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)